railties/lib/rails/application/configuration.rb

```diff
@@ -237,6 +237,12 @@ def load_defaults(target_version)
 
           if respond_to?(:action_controller)
             action_controller.raise_on_open_redirects = true
+
+            # This can't use the standard configuration pattern because you can call
+            # wrap_parameters at the top level or on any controller.
+            ActiveSupport.on_load(:action_controller) do
```

Comment:
Just for future references. Things like this should be in Railties. Here we can set a config and use that config in the `ActionController::Railtie` to execute this code. I'm moving in a follow-up commit.

Reply:
Ahhh, I should have found https://github.com/rails/rails/blob/1b0c914133a48e82832e1017ce9aa51ff021da08/actionpack/lib/action_controller/railtie.rb#L92 as an example.

I can do a followup unless you're already doing it?

Reply:
I'm already doing.

Reply:
Fixed in https://github.com/rails/rails/commit/4db42073cdb89157e0494542acb5e3fae6f55d33